awsiot/iotshadow.py

```diff
@@ -1295,15 +1303,22 @@ class ShadowState(awsiot.ModeledClass):
 
     Attributes:
         desired (typing.Dict[str, typing.Any]): The desired shadow state (from external services and devices).
+        desired_none_is_valid (bool): Set to true to allow desired to be None, clearing the data if sent.
```

Comment:
trivial: put "desired" in backticks or quotes so it's clear we're talking about the variable